src/pages/gen2/start/quickstart/nuxtjs-app/index.mdx

```diff
@@ -0,0 +1,764 @@
+                libraryOptions,
+                (contextSpec) => fetchAuthSession(contextSpec, options)
+              ),
+            fetchUserAttributes: () =>
+              runWithAmplifyServerContext(
+                amplifyConfig,
+                libraryOptions,
+                (contextSpec) => fetchUserAttributes(contextSpec)
+              ),
+            getCurrentUser: () =>
+              runWithAmplifyServerContext(
+                amplifyConfig,
+                libraryOptions,
+                (contextSpec) => getCurrentUser(contextSpec)
+              ),
+          },
+          GraphQL: {
+            client: {
+              // Follow this typing to ensure the`graphql` API return type can
+              // be inferred correctly according to your queries and mutations
+              graphql: <
+                FALLBACK_TYPES = unknown,
+                TYPED_GQL_STRING extends string = string
+              >(
+                options: GraphQLOptionsV6<FALLBACK_TYPES, TYPED_GQL_STRING>,
+                additionalHeaders?: Record<string, string>
+              ) =>
+                runWithAmplifyServerContext<
+                  GraphQLResponseV6<FALLBACK_TYPES, TYPED_GQL_STRING>
+                >(amplifyConfig, libraryOptions, (contextSpec) =>
+                  gqlServerClient.graphql(
+                    contextSpec,
+                    options,
+                    additionalHeaders
+                  )
+                ),
+            },
+          },
+        },
+      },
+    };
+  },
+});
+```
+
+The `app.vue` file can be rendered on both the client and server sides by default. The `useNuxtApp().$Amplify` composable will pick up the correct implementation of `01.amplify-apis.client.ts` and `01.amplify-apis.server.ts` to use, depending on the runtime.
+
+<Callout warning>
+
+Only a subset of Amplify APIs are usable on the server side, and as the libraries evolve, `amplify-apis.client` and `amplify-apis.server` may diverge further. You can guard your API calls to ensure an API is available in the context where you use it. E.g., you can use `if (process.client)` to ensure that a client-only API isn't executed on the server.
+
+</Callout>
+
+## Build UI
+
+Let's add UI that connects to the backend data and auth resources.
+
+### Add a login form
+
+First, install the Amplify UI component library:
+
+```bash title="Terminal" showLineNumbers={false}
+npm install @aws-amplify/ui-vue
+```
+
+Next, in your app's `app.vue` file, add the following imports and wrap your App in the `Authenticator` function:
+
+```ts title="nuxt-amplify-gen2/app.vue"
+// app.vue
+<script setup lang="ts">
+import { Authenticator } from '@aws-amplify/ui-vue';
+import '@aws-amplify/ui-vue/styles.css';
+</script>
+
+
+<template>
+  <Authenticator>
+    <template v-slot="{ user, signOut }">
+      <h1>Hello {{ user.username }}!</h1>
+      <button @click="signOut">Sign Out</button>
+    </template>
+  </Authenticator>
+</template>
+```
```

Comment:
When I copy/paste this into a fresh app following the guide, I run into this error.

<img width="1156" alt="image" src="https://github.com/aws-amplify/docs/assets/16952986/ec1e614f-291f-4884-87ae-c775fec5b040">

I had to change my tsonfig to this for it to go away:

```ts
{
  // https://nuxt.com/docs/guide/concepts/typescript
  "extends": "./.nuxt/tsconfig.json",
  "compilerOptions": {
    "moduleResolution": "Node"
  }
}
```